website/docs/components/icon/partials/code/how-to-use.md

```diff
@@ -199,10 +191,10 @@ A `prefers-reduced-motion` media query will automatically disable the animation
 
 #### In Ember apps
```

Comment:
Consider which part of this content ("Animated icons" section) needs to be moved as well to the "Getting started for Engineers > Icon" section, and what needs to be updated